src/binary-writer.h

```diff
@@ -18,6 +18,7 @@
 #define WASM_BINARY_WRITER_H_
 
 #include "common.h"
+#include "stream.h"
```

Comment:
Interestingly it was an enum I needed.. and I didn't realize I could forward declare them.

Done.